submissions/BlueLamantine/memory-pair-game/js/index.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+class General {
+  constructor(startBtn) {
+    this.startButton = document.querySelector(startBtn);
+    this.cardAreas = document.querySelectorAll('.block');
+    this.gameTheme = new Audio('./audio/game_theme.mp3');
+  }
+
+  playAudio() {
+    this.gameTheme.loop = true;
+    this.gameTheme.volume = 0.2;
+    this.gameTheme.muted = false;
+    this.gameTheme.play();
+
+    document.querySelector('#musicControl').addEventListener('click', e => {
+      e.preventDefault();
+      if (this.gameTheme.muted) {
+        this.gameTheme.muted = false;
+        e.target.classList.replace('fa-volume-off', 'fa-volume-up');
+      } else {
+        this.gameTheme.muted = true;
+        e.target.classList.replace('fa-volume-up', 'fa-volume-off');
+      }
+    });
+  }
+  setGameGonfig() {
+    this.startButton.addEventListener('click', e => {
+      e.preventDefault();
```

Comment:
why did use this `e.preventDefault();` here?

Reply:
:shrug: need to refresh my knowledge about ```e.preventDefault();```